apps/backend/src/frameworks/casl/hooks/user.hook.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { UserBeforeFilterHook } from 'nest-casl';
+
+import { DataServices } from '../../../core/abstracts/data-services.abstract';
+
+@Injectable()
+export class UserHook implements UserBeforeFilterHook<any> {
+  constructor(readonly dataServices: DataServices) {}
+
+  async run(user: any) {
+    const dbUser = await this.dataServices.users.findByEmail(user.email);
+
+    // TO DO:
+    // since nest-casl is RABC, we need to generate an array of roles.
```

Comment:
RBAC*

Also, what about filing an issue and link it to the comment so we can remember removing it in the future?

Reply:
done! https://github.com/animavita/animavita/issues/204